apps/strapi-cms/config/database.ts

```diff
@@ -1,11 +1,41 @@
 import path from 'path';
 
-export default ({ env }: any) => ({
-  connection: {
-    client: 'sqlite',
+export default ({ env }: any) => {
+  const client: string = env('DATABASE_CLIENT');
+
+  const connections: {[key: string]: any} = {
```

Comment:
```suggestion
  const connections = {
```

Reply:
That typing is used to avoid the following error

![Screenshot 2023-10-31 121426](https://github.com/pagopa/b2b-portals/assets/59611877/dc5772ed-491e-4bf8-90ea-9d5cabb1e78d)

Reply:
Yes I knew, this is why I've suggested also [this one](https://github.com/pagopa/b2b-portals/pull/50#discussion_r1377381557) :)